index.html

```diff
@@ -386,15 +384,21 @@ <h3>Execute Multi-stage Presentation Workflow</h3>
               Shipping Co. Website - Issuer App->>Kirk Company Enterprise Resource Planning App - Holder and Holder Service:kirkcompany.com/credentails/newCredential?\"BoL\"     
               Kirk Company Enterprise Resource Planning App - Holder and Holder Service->>Ignio - Logistic Manager:BoL Ready notice 
             </pre>
-            <figcaption>UC2.3 Mermaid</figcaption>
+            <figcaption>UC6.3 Mermaid</figcaption>
           </figure>
         </p>
       <p><i>Contributed by: </i>Digital Bazaar</p>
   </section>
 
   <section>
     <h3>Aggregated Credential Workflow</h3>
-    <p>Tod is using digital credentials to apply for a service/qualify for a job/gain access to a programme. 
+    <p>Kenzie is using digital credentials to apply for a job at as a teacher. 
+      Her teaching certification is already in her digital wallet but the school also requires a 
+      criminal background check which Kenzie does not currently have a credential for. The process
+      of getting the background check can take 24-48 hours. Kenzie goes to the job application
+      website and submits her application, the website responds requesting both her teaching credential
```

Comment:
```suggestion
      website and submits her application; the website responds requesting both her teaching credential
```

Reply:
@TallTed Thanks for all the formatting suggestions and grammar clarifications. Just wanted you to know that they should be resolved in the next week, I am working on some updates to our script that outputs the mermaid to include some of them automatically so (e.g. the <br> tags) so I don't have to add those manually in the future.

Reply:
Thanks for the update, @eric-schuh -- and glad I didn't leap into adding all the `<br>` tags! I hadn't absorbed that the mermaid was being generated somewhere, rather than being a major hand-craft. Might be a good thing to add a notice of this before and/or after the mermaid blocks in the documents, so nobody else edits them directly!